src/main/java/com/hivemq/mqtt/topic/SubscriberWithQoS.java

```diff
@@ -27,15 +27,13 @@
 
 /**
  * This represents a subscriber (client ID) with a Quality of Service Level
- *
- * @author Dominik Obermaier
```

Comment:
I'm also not sure if we wanted to remove the authors form the CE.

Reply:
@SgtSilvio do you happen to know this?

Reply:
@LBrandl Silvio told me that the rule of not removing authors from APIs and removing them from impls applies to all of our projects, including CE